src/corelibs/U2Formats/src/PDBFormat.cpp

```diff
@@ -426,7 +426,11 @@ void PDBFormat::PDBParser::parseAtom(BioStruct3D &biostruct, U2OpStatus &) {
     biostruct.modelMap[currentModelIndex + 1].insert(id, a);
 
     if (atomIsInChain) {
+        SAFE_POINT_EXT(0 <= chainIndex && chainIndex < biostruct.moleculeMap.size(), os.setError("Unexpected chainIndex"), );
```

Comment:
tr()?

Reply:
Historically, I've never translated SAFE_POINTs. Should i?